src/defaults.ts

```diff
@@ -17,9 +17,9 @@ export function _getDefaults(): MarkedOptions {
     walkTokens: null,
   };
 }
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export let _defaults: MarkedOptions<any, any> = _getDefaults();
```

Comment:
Why `any`? Is that because we don't want to pass in generic types to _getDefaults()?